packages/angular_devkit/build_angular/test/karma/rebuilds_spec_large.ts

```diff
@@ -64,4 +65,9 @@ xdescribe('Karma Builder watch mode', () => {
       take(3),
     ).toPromise().then(done, done.fail);
   }, 30000);
+
+  it('does not rebuild when nothing changed', (done) => {
+    // Start the server in watch mode, wait for the first build to finish, touch
+    // test.js without changing it, wait 5s then exit unsuscribe, verify only one event was emitted.
+  }, 30000);
```

Comment:
There's no code in here.  is that intentional?

Reply:
Yeap, see the comment above. We can't actually test karma in watch mode because there's no way for the karma server to exit gracefully without killing the processes.

Reply:
This at least needs a comment, anyone looking at this code later will be very confused why it claims to test something and then doesn't.
Did you think about another way to test this?